tests/end_to_end/test_helper.cpp

```diff
@@ -315,12 +315,15 @@ _test_helper_end_to_end::_test_helper_end_to_end()
 
 _test_helper_end_to_end::~_test_helper_end_to_end()
 {
-    // Run down duplicate handles, if any.
-    _duplicate_handles.rundown();
-    // Verify that all maps were successfully removed.
-    uint32_t id;
-    REQUIRE(bpf_map_get_next_id(0, &id) < 0);
-    REQUIRE(errno == ENOENT);
+    try {
+        // Run down duplicate handles, if any.
+        _duplicate_handles.rundown();
+        // Verify that all maps were successfully removed.
+        uint32_t id;
+        REQUIRE(bpf_map_get_next_id(0, &id) < 0);
+        REQUIRE(errno == ENOENT);
+    } catch (...) {
+    }
```

Comment:
catching all exceptions (...) is dangerous since it can leave things in an indeterminate state if say a stack exception is caught.

Reply:
Fixed.